src/Controller/Project/CcaController.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+namespace App\Controller\Project;
+
+use App\Entity\Project\Cca;
+use App\Entity\Project\Ce;
+use App\Form\Project\CcaType;
+use App\Form\Project\SubCcaType;
+use App\Service\Project\DocTypeManager;
+use DateTime;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Form\Extension\Core\Type\HiddenType;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Doctrine\ORM\EntityManagerInterface;
+
+class CcaController extends AbstractController
+{
+    protected $em;
+
+    public function __construct(EntityManagerInterface $em)
+    {
+        $this->em = $em;
+    }
+
+    /**
+     * @Security("has_role('ROLE_SUIVEUR')")
+     * @Route("/suivi/cca", name="project_cca_index")
+     *
+     * @return RedirectResponse|Response
+     */
+    public function index(): Response
+    {
+        $em = $this->em;
+
+        $ccas = $em->getRepository(Cca::class)->findAll();
+
+        return $this->render('Project/Cca/index.html.twig', [
+            'controller_name' => 'CcaController',
+            'ccas' => $ccas,
+        ]);
+    }
+
+    /**
+     * @Security("has_role('ROLE_SUIVEUR')")
+     * @Route("/suivi/cca/voir/{id}", name="project_cca_voir")
+     *
+     * @return RedirectResponse|Response
+     */
+    public function voir(Cca $cca): Response
+    {
+        return $this->render('Project/Cca/voir.html.twig', [
+            'controller_name' => 'CcaController',
+            'cca' => $cca,
+        ]);
+    }
+
+    /**
+     * @Security("has_role('ROLE_SUIVEUR')")
+     * @Route("/suivi/cca/bdc/{id}", name="project_cca_bdc")
+     *
+     * @return RedirectResponse|Response
+     */
+    public function voirBdc(Cca $cca): Response
+    {
+        return $this->render('Project/Cca/bdcs.html.twig', [
+            'controller_name' => 'CcaController',
+            'cca' => $cca,
+        ]);
+    }
+
+    /**
+     * @Security("has_role('ROLE_CA')")
+     * @Route("/suivi/cca/delete/{id}", name="project_cca_supprimer")
+     *
+     * @return RedirectResponse
+     */
+    public function supprimer(Cca $cca, Request $request): Response
+    {
+        $em = $this->em;
+
+        $form = $this->createDeleteForm($cca);
+        $form->handleRequest($request);
+
+        if ($form->isValid()) {
+            $nomProspect = $cca->getProspect()->getNom();
+            $em->remove($cca);
+            $em->flush($cca);
+
+            $this->addFlash('success', 'Convention Cadre Agile avec ' . $nomProspect . ' bien supprimée');
```

Comment:
Wording: la plupart des messages de suppressions sont sur le format "XXX supprimé". Autant rester homogène

```suggestion
            $this->addFlash('success', 'Convention Cadre Agile avec ' . $nomProspect . ' supprimée');
```